src/Components/TodoItem.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+function TodoItem(props) {
+  return (
+    <li>
+      <input
+        type={"checkbox"}
+        checked={props.item.completed}
+        onChange={() => props.changeEvent(props.item.id)}
+      />
+      {props.item.title}
+      <button onClick={() => props.deleteItem(props.item.id)}>Delete</button>
+    </li>
```

Comment:
[Part 4 – How to edit to dos items](https://ibaslogic.com/how-to-edit-todos-items-in-react/). is missing, please complete this section so that the todo items are editable